cluster/src/main/java/com/hartwig/pipeline/datatypes/DataType.java

```diff
@@ -5,24 +5,25 @@ public enum DataType {
     ALIGNED_READS_INDEX,
     WGSMETRICS,
     GERMLINE_VARIANTS,
-    GERMLINE_VARIANTS_SAGE,
+    SAGE_GERMLINE_VARIANTS,
     GERMLINE_GENE_COVERAGE,
     GERMLINE_TUMOR_SAMPLE_BQR_PLOT,
     GERMLINE_REF_SAMPLE_BQR_PLOT,
     SOMATIC_GENE_COVERAGE,
-    SOMATIC_VARIANTS_SAGE,
+    SAGE_SOMATIC_VARIANTS,
     SOMATIC_TUMOR_SAMPLE_BQR_PLOT,
     SOMATIC_REF_SAMPLE_BQR_PLOT,
     STRUCTURAL_VARIANTS_GRIDSS,
-    SOMATIC_STRUCTURAL_VARIANTS_GRIPSS_RECOVERY,
-    SOMATIC_STRUCTURAL_VARIANTS_GRIPSS,
-    GERMLINE_STRUCTURAL_VARIANTS_GRIPSS_RECOVERY,
-    GERMLINE_STRUCTURAL_VARIANTS_GRIPSS,
-    GERMLINE_VARIANTS_PAVE,
-    SOMATIC_VARIANTS_PAVE,
-    GERMLINE_VARIANTS_PURPLE,
-    SOMATIC_VARIANTS_PURPLE,
-    STRUCTURAL_VARIANTS_PURPLE,
+    GRIPSS_SOMATIC_STRUCTURAL_VARIANTS_UNFILTERED,
+    GRIPSS_SOMATIC_STRUCTURAL_VARIANTS,
+    GRIPSS_GERMLINE_STRUCTURAL_VARIANTS_UNFILTERED,
+    GRIPSS_GERMLINE_STRUCTURAL_VARIANTS,
+    PAVE_GERMLINE_VARIANTS,
+    PAVE_SOMATIC_VARIANTS,
+    PURPLE_GERMLINE_VARIANTS,
+    PURPLE_SOMATIC_VARIANTS,
+    PURPLE_SOMATIC_STRUCTURAL_VARIANTS,
+    PURPLE_GERMLINE_STRUCTURAL_VARIANTS,
```

Comment:
These really do read better I think, but unfortunately this represents quite a big change as these enumeration constants are what we use to actually populate the datasets.

Put another way, while these are constants in the pipeline code and renaming them doesn't break anything here, their names are used to populate our API database. There won't be a non-zero downstream consequence here so we should be deliberate with changing these. Do they need to change?

Reply:
They were already inconsistent as you can see and should remain changed. Sorry for the additional downstream work.